content/docs/guides/crossguard/core-concepts.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Core Concepts
+linktitle: Core Concepts
+
+menu:
+  userguides:
+    parent: crossguard
+    identifier: crossguard-core-concepts
+---
+{{% crossguard-preview %}}
+
+## Policy
+
+A Policy contains specific logic you would like to enforce. For example, you may want to restrict the creation of public S3 buckets or you may disallow resource provisioning without tags. You can refer to other examples [here](https://github.com/pulumi/examples/tree/master/policy-packs).
```

Comment:
"creation of a public S3 bucket" assumes the reader is familiar with AWS and AWS-isms. Perhaps we should be more general here?

"For example, you may want to prevent the creation of public, world-readable storage objects. (e.g. on AWS S3, Azure BlobStore, etc.) Or prevent the creation of a virtual machine without the proper security groups in-place."

Nit: I'd also remove the "refer to other examples", and just put that later. As linking to examples might be a distraction if this page is focused on the core concept. Instead, just put it at the end like "You can look at our Pulumi Policy examples repositories to see examples of `ResourceValidationPolicy` and `StackValidationPolicy` rules for common cloud providers.